genai-on-vertex-ai/genai-website-mod/backend/models/vertex_llm_models.py

```diff
@@ -17,7 +17,7 @@
 
 class VertexLLMRequest(BaseModel):
     prompt: str
-    model_name: str = "text-bison"
+    model_name: str = "gemini-pro"
```

Comment:
Let's add the fully qualified model endpoint - `gemini-1.5-pro-001` etc, to ensure more accurate tracability from user perspective.